src/community/wmts-multi-dimensional/src/main/java/org/geoserver/gwc/wmts/dimensions/RasterElevationDimension.java

```diff
@@ -0,0 +1,50 @@
+/* (c) 2016 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.gwc.wmts.dimensions;
+
+import org.geoserver.catalog.*;
+import org.geoserver.catalog.util.ReaderDimensionsAccessor;
+import org.geoserver.wms.WMS;
+import org.geotools.coverage.grid.io.GridCoverage2DReader;
+import org.opengis.filter.Filter;
+
+import java.io.IOException;
+import java.util.TreeSet;
+
+/**
+ * Represents an elevation dimension of a raster.
+ */
+public class RasterElevationDimension extends Dimension {
```

Comment:
This whole family of classes could use a toString()  implementation, it's harder to debug without